test/evals/eval_data.yaml

```diff
@@ -166,8 +166,8 @@
   conversation:
     - eval_id: refuse_azure_request
       eval_query: Can you create a cluster on Azure?
-      eval_types: [response_eval:sub-string]
-      expected_keywords: ["Azure", "cannot create", "Assisted Installer", "on-prem"]
+      eval_types: [response_eval:accuracy]
+      expected_response: Sorry, I can't create clusters directly on public clouds like Azure. I am designed to assist with on-premise OpenShift installations using the Assisted Installer. For installing OpenShift on Azure, you can explore Red Hat OpenShift Service on Azure (ROSA) or use the standard OpenShift Installer.
```

Comment:
I believe ROSA is AWS and ARO is Azure.
ref: https://www.redhat.com/en/blog/creating-azure-red-hat-openshift-and-red-hat-openshift-services-on-aws-clusters-declaratively